daemon/algod/api/server/v2/dryrun.go

```diff
@@ -397,17 +397,22 @@ func doDryrunRequest(dr *DryrunRequest, response *generated.DryrunResponse) {
 		return
 	}
 	proto := config.Consensus[protocol.ConsensusVersion(dr.ProtocolVersion)]
-
+	proto.EnableAppCostPooling = true
+	proto.MaxTxGroupSize = 16
 	response.Txns = make([]generated.DryrunTxnResult, len(dr.Txns))
 	for ti, stxn := range dr.Txns {
 		pse := logic.MakePastSideEffects(len(dr.Txns))
+		maxBudget := uint64(proto.MaxAppProgramCost * proto.MaxTxGroupSize)
+		evalBudget := uint64(proto.MaxAppProgramCost * proto.MaxTxGroupSize)
```

Comment:
I think it's a slightly clearer thing to say `evalBudget := maxBudget` rather than repeat the init expression.  They are not two variables that happens to be set the same. evalBudget is a temporary value that must begin as maxBudget.